scripts/check_integration.sh

```diff
@@ -16,6 +16,6 @@ echo "Running adapter integration tests with tag=$TAGS"
 go test -count=1 -v -cover . -p 1 -timeout=40m $TAGS
 
 echo "Running adapter didcomm v2 integration tests"
-AGENT_KEY_TYPE=ecdsap256ieee1363 AGENT_KEY_AGREEMENT_TYPE=p256kw USE_DIDCOMM_V2=true AGENT_MEDIA_TYPE_PROFILES=didcomm/v2 go test -count=1 -v -cover . -p 1 -timeout=40m -run waci_didcommv2
+go test -count=1 -v -cover . -p 1 -timeout=40m -run waci_didcommv2
```

Comment:
@Moopli Do we need to run this separately now that we have merged both into one ?

Reply:
We'd need to modify the tests to clear connection data from the containers, it's just simpler to start them fresh